README.md

```
@@ -68,6 +68,81 @@
                 Implementations (Prisma, etc.)
 ```
 
+### アーキテクチャ図
+
+MUED LMSはマイクロサービスアーキテクチャを採用しており、以下の主要サービスで構成されています：
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)